spec/src/main/asciidoc/WebServicesMetadata.adoc

```diff
@@ -943,2259 +807,1399 @@ header, and the mode is OUT.
+          <handler-class>fromwsdl.handler.common.BaseLogicalHandler</handler-class>
+          <init-param>
+            <param-name>handlerName</param-name>
+            <param-value>client0</param-value>
+          </init-param>
+        </handler>
+      </handler-chain>
+      <handler-chain>
+        <port-name-pattern xmlns:ns2="urn:test">ns2:Report</port-name-pattern>
+        <handler>
+          <handler-class>fromwsdl.handler.common.BaseLogicalHandler</handler-class>
+          <init-param>
+            <param-name>handlerName</param-name>
+            <param-value>client2</param-value>
+          </init-param>
+        </handler>
+      </handler-chain>
+      <handler-chain>
+        <port-name-pattern xmlns:ns2="urn:test">ns2:ReportServicePort</port-name-pattern>
+        <handler>
+          <handler-class>fromwsdl.handler.common.BaseSOAPHandler</handler-class>
+          <init-param>
+            <param-name>handlerName</param-name>
+            <param-value>client6</param-value>
+          </init-param>
+        </handler>
+      </handler-chain>
+      <handler-chain>
+        <protocol-bindings>##SOAP11_HTTP</protocol-bindings>
+        <handler>
+          <handler-class>fromwsdl.handler.common.BaseSOAPHandler</handler-class>
+          <init-param>
+            <param-name>handlerName</param-name>
+            <param-value>client7</param-value>
+          </init-param>
+          <soap-role>http://sun.com/client/role1</soap-role>
+          <soap-role>http://sun.com/client/role2</soap-role>
+        </handler>
+      </handler-chain>
+      <handler-chain>
+        <protocol-bindings>##SOAP11_HTTP</protocol-bindings>
+        <handler>
+          <handler-class>fromwsdl.handler.common.BaseLogicalHandler</handler-class>
+          <init-param>
+            <param-name>handlerName</param-name>
+            <param-value>client3</param-value>
+          </init-param>
+        </handler>
+      </handler-chain>
+    </handler-chains>
+  </bindings>
+</bindings
+----
 
-<operation name="PingTwoWay">
+=== Annotation: javax.jws.soap.SOAPBinding
 
-<input message="tns:PingTwoWay"/>
+==== Description
 
-<output message="tns:PingTwoWayResponse"/>
+Specifies the mapping of the Web Service onto the SOAP message protocol.
+Section _6 SOAP Binding_ describes the effects of this annotation on
+generated Web Services. The SOAPBinding annotation has a target of TYPE
+and METHOD. The annotation may be placed on a method if and only if the
+SOAPBinding.style is DOCUMENT. Implementations MUST report an error if
+the SOAPBinding annotation is placed on a method with a
+SOAPBinding.style of RPC. Methods that do not have a SOAPBinding
+annotation accept the SOAPBinding behavior defined on the type.
 
-</operation>
+The @SOAPBinding annotation includes the following member-value pairs.
 
-<operation name="SecurePing">
+[cols=3, options=header]
+|===
+|Member-Value
+|Meaning
+|Default
 
-<input message="tns:SecurePing"/> +
-</operation>
+|Style
```

Comment:
should start with lower case - _style_

Reply:
ok, resolved by https://github.com/eclipse-ee4j/jws-api/pull/69/commits/62fdf397f28a148826367da36bcaef2c30e97c7d